### PR TITLE
Guess who - Maria Pogosjan 

### PR DESCRIPTION
**Pull request by MariaPogosjan**
Hi! 

**As you can see I kept the same layout. However I made some changes with the css:** 
- I made different background colors in checkMyGuess, depending if the users guesses right (green) or wrong (red).
- Made some buttons smaller
- Added some small changes so it would look better in mobile size.  
- I also added <script src="https://unpkg.com/sweetalert/dist/sweetalert.min.js"> in my HTML to be able to use Sweetalert's custom alert messages. Therefore on my JS I don't have "alert" in some places, instead "swal"

**Problems I couldn't solve:** 
- The timer starts from the beginning when the prompt message appairs, I couldn't manage to solve that. 
- The sound effects have some delay on mobile version, why?

### CODE REVIEW

**Review by klimenko3007 — CHANGES_REQUESTED** [review]
I think you did a fantastic job using a lot of advanced code we have not learned about! I am really impressed! Cool sound effects! 
I think though it is important to change the wrongGuess value in your start function. Otherwise the game does not work how it should.

File: code/script.js [hunk 1]
```diff
@@ -249,95 +339,238 @@ const setSecret = () => {
...
+  winOrLose.style.display = 'none'
+  document.querySelector('.timer').style.display = 'flex'
+
+  guessCount = 0
+  seconds = 0 
+  minutes = 0
+  generateBoard()
+  setSecret()
+  showGuessCount()
 }
```
[the remark this answers is not in the document]
Comment (MariaPogosjan) [reply code/script.js:378]: Thank you for noticing it! I have accidentally deleted the wrongGuess from start. I added it back and it works now.

File: code/style.css [hunk 2]
```diff
@@ -230,4 +250,22 @@ button {
...
+    top: 0;
+    position: fixed;
+    z-index: 1;
+    width: 100%;
+    height: 5px;
+    background-color: var(--secondary);
+    padding: 3px 20px 20px 10px;
+    font-weight: 400;
+    font-size: 12px;
+  }
```
[the remark this answers is not in the document]
Comment (MariaPogosjan) [reply code/style.css:270]: I tried to look at this but I can't see that the numbers are jumping on big screen or mobile. I looked a both but can't find the same problem. Could you describe where you see this? :)

File: code/script.js [hunk 3]
```diff
@@ -10,80 +15,107 @@ const CHARACTERS = [
     img: 'images/jabala.svg',
     hairColor: 'hidden',
     eyeColor: 'hidden',
+    woman: false,
+    man: true,
```
Comment (KarolCislowski) [inline code/script.js:19]: Nice that you decide to experiment with the given data 🥼

File: code/script.js [hunk 4]
```diff
@@ -249,95 +339,238 @@ const setSecret = () => {
...
+  board.style.display = 'flex'
+  winOrLose.style.display = 'none'
+  document.querySelector('.timer').style.display = 'flex'
+
+  guessCount = 0
+  seconds = 0 
+  minutes = 0
+  generateBoard()
+  setSecret()
+  showGuessCount()
```
Comment (KarolCislowski) [inline code/script.js:377]: you can try to move your timer here after all alerts already happend 🤓

File: code/style.css (hunk 2 above)
[the remark this answers is not in the document]
Comment (KarolCislowski) [reply code/style.css:270]: `overflow: hidden;` should make a trick with hiding scrollbars 🤓

**Review by KarolCislowski — APPROVED** [review]
This is a very impressive project! 🌈  You have shown great ambition 🦄 

I think the issue with the sound on mobile phones is due to the different access to the sound system. You can play sounds at once on the computer, and mobiles need to queue them.

**Pull request closed without merging**